config/logging.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
Any effects we should prepare for in terms of log file generation? Any different paths / locations?

Reply:
AFAIK, I don't think there are any changes to be made.